src/utils/parser/utils/ast.js

```diff
@@ -15,7 +15,20 @@ function _parse(code, opts) {
     code,
     Object.assign({}, opts, {
       sourceType: "module",
-      plugins: ["jsx", "flow"]
+      plugins: [
```

Comment:
There's a plugin for "decorators" that I could enable but it's based on a rejected version of the language feature proposal, the rest here are not rejected.